mine/models/stg_order_items.sql

```diff
@@ -0,0 +1,16 @@
+with source as ( select {{ dbt_utils.surrogate_key(['order_id', 'order_item_id']) }} as order_item_sk
```

Comment:
the SK is important to add, but I would add it in the fixed CTE. The source CTE is generally a pass through that doesn't actually add any modelling